app/pages/projects/[projectId]/index.tsx

```diff
@@ -336,10 +336,12 @@ export const Project = () => {
                 >
                   {member?.active ? "Suspend my Membership" : "Join Project Again"}
                 </Button>
-              ) : (
+              ) : project.helpWanted ? (
```

Comment:
Could be more clean in this way :
project.helpWanted && (
```suggestion
              ) : project.helpWanted ? (
```
                <Button className="primary large" onClick={handleJoinProject}>
                  Want to Join?
                </Button> )